app/main.ipynb

```diff
@@ -31,17 +33,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# write your code for task 0 here"
+    "df = pd.read_csv(r\"C:\\Users\\Alex\\py-adult-data-analysis\\data\\adult.csv\")\n",
+    "df = df.replace(\"?\", None)\n",
+    "df = df.dropna()\n",
+    "df"
```

Comment:
you should check data for correctness. So if salary = "<=50K", then you should ensure that column "salary K$" has an appropriate value